docs/postgres.rst

```diff
@@ -28,6 +28,7 @@ virtual env::
 
     sudo apt-get install libpq-dev
     pip install psycopg2
+    pip install psycopg2-binary
```

Comment:
as I understand it `psycopg2` and `psycopg2-binary` are alternatives - why would we need both?

(and even if we do need both, why do they need to be installed with separate `pip` invocations?)